Controller/Admin/TransactionTabPostProcessing.php

```diff
@@ -98,15 +134,40 @@ private function _validateRequest(array $aRequestParameters)
     {
         $fAmount = $aRequestParameters['amount'];
 
-        if (!$fAmount || !is_numeric($fAmount)) {
+        if (!$this->_validateAmountIsNumeric($fAmount)) {
             throw new Exception(Helper::translate('text_generic_error'));
         }
 
-        if ($fAmount < 0 || $fAmount > $this->oTransaction->wdoxidee_ordertransactions__amount->value) {
+        if (!$this->_validateAmountInRange($fAmount, $this->oTransaction->wdoxidee_ordertransactions__amount->value)) {
             throw new Exception(Helper::translate('total_amount_not_in_range_text'));
         }
     }
 
+    /**
+     * Checks that the amount argument is a numeric value
+     *
+     * @param float $fAmount
+     *
+     * @return boolean true/false whether amount argument is a numeric value
+     */
+    private function _validateAmountIsNumeric($fAmount)
```

Comment:
if i give the method for example '3.222,12' will i break the code. I think so, in paymentSDK we do return is_float($value) || is_int($value)

Reply:
Thanks for the hint, changed it to the same validation as in the paymentSDK.